hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -255,8 +264,14 @@ private FlinkOptions() {
   public static final ConfigOption<Double> WRITE_BATCH_SIZE = ConfigOptions
       .key("write.batch.size.MB")
       .doubleType()
-      .defaultValue(2D) // 2MB
-      .withDescription("Batch buffer size in MB to flush data into the underneath filesystem");
+      .defaultValue(64D) // 64MB
+      .withDescription("Batch buffer size in MB to flush data into the underneath filesystem, default 64MB");
+
+  public static final ConfigOption<Integer> WRITE_LOG_BLOCK_SIZE = ConfigOptions
+      .key("write.log_block.size")
```

Comment:
Can we also add a unit like `WRITE_BATCH_SIZE `?

Reply:
No, i found the unit is verbose. People should see the comment in the document if they want to use this config option.

Reply:
IMO, between verbosity and following a uniform style, I think the latter is more important and allows users to feel a consistent experience.